.github/workflows/go.yml

```diff
@@ -5,10 +5,18 @@ on:
       - master
       - main
   pull_request:
+    paths:
+      - '**.go'
+      - '**go.mod'
+      - '**go.sum'
 
 permissions:
   contents: read
 
+env:
+  GO_VERSION: '1.22.1'
```

Comment:
Should we upgrade? Any reason why we are at 1.22.x and not 1.23.x ?

Reply:
We could indeed bump the go version. Should be done in another PR.